### PR TITLE
Upgrade the manual to docbook 5.0

### PR DESCRIPTION
**Pull request by rbuj**
To upgrade the manual:
sudo dnf install -y docbook5-schemas
xsltproc --output index-new.docbook /usr/share/xml/docbook5/stylesheet/upgrade/db4-upgrade.xsl index.docbook
xsltproc --output legal-new.xml /usr/share/xml/docbook5/stylesheet/upgrade/db4-upgrade.xsl legal.xml
(...)

To validate the manual:
xmllint --noout --relaxng /usr/share/xml/docbook5/schema/rng/5.0/docbookxi.rng mate-user-guide/C/index.docbook
jing /usr/share/xml/docbook5/schema/rng/5.0/docbookxi.rng mate-user-guide/C/index.docbook
yelp-check validate mate-user-guide/C/index.docbook

To view the manual:
yelp file:///full_path/index.docbook

Note: docbook5-schemas should be installed in your system in order to view the manual (DEPENDENCY)

### CODE REVIEW

**Comment by rbuj** [comment]
yelp-check can't validate index.docbook, and yelp-build produces empty docs.
highlights element [was removed from DocBook 5](https://docbook.org/docs/howto/howto.html#changes-removed), and validation error for [anchor](https://tdg.docbook.org/tdg/5.0/anchor.html) as child of [chapter](https://tdg.docbook.org/tdg/5.0/chapter.html).

File: mate-user-guide/C/index.docbook [hunk 1]
```diff
@@ -245,26 +261,20 @@
       </revision>
     </revhistory>
 
-    <releaseinfo>This manual describes version &mateversion; of the
-    MATE desktop.</releaseinfo>
+    <releaseinfo>This manual describes version 1.10 of the MATE desktop.</releaseinfo>
```
Comment (raveit65) [inline mate-user-guide/C/index.docbook]: Can we update MATE version please?

**Comment by raveit65** [comment]
@rbuj 
I think we can merge that if version is fixed.

**Comment by rbuj** [comment]
@raveit65
done

**Pull request merged**